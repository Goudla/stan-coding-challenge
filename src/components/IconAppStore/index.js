import * as React from 'react';

// Assets
import appStoreSVG from './assets/app-store.svg';

class IconAppStore extends React.PureComponent<*> {
  render() {
    return (
      <img src={appStoreSVG} alt="App Store" width="135" />
    );
  }
}

export default IconAppStore;
