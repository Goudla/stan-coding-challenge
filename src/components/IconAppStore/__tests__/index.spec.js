import React from 'react';
import ShallowRenderer from 'react-test-renderer/shallow';

// Scenes
import IconAppStore from '../';

describe('<IconAppStore />', () => {
  it('Should render', () => {
    const renderer = new ShallowRenderer();
    renderer.render(<IconAppStore />);
    const result = renderer.getRenderOutput();
    expect(result).toMatchSnapshot();
  });
});
