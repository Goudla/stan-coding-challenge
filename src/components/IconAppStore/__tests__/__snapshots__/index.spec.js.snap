// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<IconAppStore /> Should render 1`] = `
<img
  alt="App Store"
  src="app-store.svg"
  width="135"
/>
`;
